README.md

```diff
@@ -99,4 +104,3 @@ See [CONTRIBUTING.md](./CONTRIBUTING.md).
 ## Credits
 
 `rustlings` was originally written by [Carol](https://github.com/carols10cents)!
-
```

Comment:
The line ending at the end here would need to be readded, but the rest looks good :+1: